dag.js

```diff
@@ -3,11 +3,12 @@ import ReactDOM from 'react-dom';
 import {configureStore} from './dag-store';
 import {getSettings} from './dag-settings';
 import uuid from 'node-uuid';
+import jsPlumb from 'jsPlumb';
```

Comment:
No worries, they need to make a law that any UI library needs to work with Webpack going forward ;)